R/orderly.R

```diff
@@ -21,7 +21,10 @@ orderly_list_src <- function(root = NULL, locate = TRUE) {
     return(character())
   }
   pos <- fs::dir_ls(file.path(root_path, "src"), type = "directory")
-  basename(pos)[file_exists(file.path(pos, "orderly.R"))]
+  file_paths <- lapply(pos, function(p) {
```

Comment:
```suggestion
  file_paths <- vcapply(pos, function(p) {
```

map returning character vector I think?

Reply:
`file_paths <- file.path(p, get_orderly_file_names(p))` works as well